packages/reaction-email-templates/templates.js

```diff
@@ -1,40 +0,0 @@
-/**
- * ReactionEmailTemplate - Returns a template source for SSR consumption
- * layout must be defined + template
- * @param {String} template name of the template in either Layouts or fs
- * @returns {Object} returns source
- */
-ReactionEmailTemplate = function (template) {
-  check(template, String);
-  let source;
-  let lang = "en";
-
-  const shopLocale = Meteor.call("shop/getLocale");
-
-  if (shopLocale && shopLocale.locale && shopLocale.locale.languages) {
-    lang = shopLocale.locale.languages;
-  }
-
-  // using layout where in the future a more comprehensive rule based
-  // filter of the email templates can be implemented.
-  const tpl = ReactionCore.Collections.Packages.findOne({
-    "layout.template": template
-  });
-
-  if (tpl) {
```

Comment:
Ok, was not sure if the code is really working and/or in use. Looked more like a not-finished-thing to me. 

Let me try to re-integrate it.